extension/js/common/composer.ts

```diff
@@ -1536,6 +1537,15 @@ export class Composer {
       }, 1000);
     } else {
       $('.close_new_message').click(Ui.event.handle(() => this.app.closeMsg(), this.getErrHandlers(`close message`)));
+      $('.minimize_new_message').click(Ui.event.handle(() => {
+        let height = '36px';
+        if (this.composeWindowIsMinimized) {
+          height = '605px';
+        }
+        BrowserMsg.send.setCss(this.v.parentTabId, { selector: `div#new_message`, css: { height } });
+        BrowserMsg.send.setCss(this.v.parentTabId, { selector: `iframe#${this.v.frameId}`, css: { height } });
```

Comment:
Looks good! You could probably simplify all of this it into a single call, with:
```js
BrowserMsg.send.setCss(this.v.parentTabId, { selector: `iframe#${this.v.frameId}, div#new_message`, css: { height: this.composeWindowIsMinimized ? '605px' : '36px' } });
```
(please test to confirm, I think it should take two selectors without issue)